frontend/js/main.js

```diff
@@ -0,0 +1,2203 @@
+/**
+ * Function to initiate home screen tasks like validating form input fields
+ *
+ * @return {null}
+ */
+const initHomeScreenTasks = () => {
+  $(document).ready(function () {
+    setSessionTableContent();
+    $('#loadDbForm > div').keyup(function () {
+      var empty = false;
+      $('#loadDbForm > div > input').each(function () {
+        if ($(this).val() == '') {
+          empty = true;
+        }
+      });
+      if (empty) {
+        $('#loadConnectButton').attr('disabled', 'disabled');
+      } else {
+        $('#loadConnectButton').removeAttr('disabled');
+      }
+    });
+    $('#connectForm > div').keyup(function () {
+      var empty = false;
+      $('#connectForm > div > input').each(function () {
+        if ($(this).val() == '') {
+          empty = true;
+        }
+      });
+      if (empty) {
+        $('#connectButton').attr('disabled', 'disabled');
+      }
+      else {
+        $('#connectButton').removeAttr('disabled');
+      }
+    });
+  })
+}
+
+/**
+ * Function to trigger click event while file uploading
+ *
+ * @return {null}
+ */
+$(function () {
+  $("#upload_link").on('click', function (e) {
+    e.preventDefault();
+    $("#upload:hidden").trigger('click');
+  });
+});
+
+/**
+ * Function to update selected file name while file uploading
+ *
+ * @return {null}
+ */
+$(document).ready(function () {
+  $('#upload').change(function () {
+    fileName = $('#upload')[0].files[0].name;
+    if (fileName != '') {
+      $('#importButton').removeAttr('disabled');
+    }
+    $("#upload_link").text(fileName);
+  });
+});
+
+/**
+ * Function to read the json content of selected file while file uploading
+ *
+ * @return {null}
+ */
+$(document).on('change', '#upload', function (event) {
+  var reader = new FileReader();
+  reader.onload = function (event) {
+    var jsonObj = JSON.parse(event.target.result);
+    importSchemaObj = jsonObj;
```

Comment:
Local variable 'jsonObj' is redundant.

`importSchemaObj = JSON.parse(event.target.result);`

Reply:
Agreed.. code changes done accordingly